tests/test_istio.py

```diff
@@ -25,7 +25,7 @@ def test_istio(self):
         Sets up and validate istio.
         """
         print("Enabling Istio")
-        microk8s_enable("istio")
+        microk8s_enable("istio", 300, optional_args="--istio-version 1.20.0")
```

Comment:
I'd suggest we still keep the testcase with the default istio version which is in `addons.yaml` so both can be tested.

Could you please [`pytest.mark.parametrize`](https://docs.pytest.org/en/stable/how-to/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions) the testcase so it takes a list of istio versions we'd like to test?

Maybe include `None` or `""` to test the default value from `addons.yaml` too?